nck/readers/dbm_reader.py

```diff
@@ -171,7 +174,7 @@ def get_query_report_url(self, existing_query=True):
             query_id = query_infos["queryId"]
             query_infos = self._wait_for_query(query_id)
 
-        if query_infos["metadata"]["googleCloudStoragePathForLatestReport"]:
+        if query_infos["metadata"].get("googleCloudStoragePathForLatestReport", None):
```

Comment:
What about:

```python
if "googleCloudStoragePathForLatestReport" in query_infos["metadata"]:
    url = query_infos["metadata"]["googleCloudStoragePathForLatestReport"]
```

instead.

I think this is the behavior you want and this is slightly more readable.

Reply:
Hm unless the case where the key is present but the value empty can happen. Then you need something like:

```python
if (
    "googleCloudStoragePathForLatestReport" in query_infos["metadata"] and len(query_infos["metadata"]. 
   ["googleCloudStoragePathForLatestReport"]) > 0
):
    url = query_infos["metadata"]["googleCloudStoragePathForLatestReport"]
```

This is more explicit that the `get(my_field, None)`.

Reply:
It was the behavior with DBM v1. Now this is a bit weird with DBM v1.1 and apparently there is now only one key between "googleCloudStoragePathForLatestReport" and "googleDrivePathForLatestReport".

And now, the variable query_infos["metadata"]["running"] (which purpose is to tell if the query/report is running or not) is not working very well with DBM v1.1.
For example, if your report is ready (i.e query_infos["metadata"]["running"] == False), you sometimes have to wait a little bit more to retrieve the url thanks to the key "googleCloudStoragePathForLatestReport" because sometimes the key is not in the API response directly.

Reply:
Ok thank you for the context. Very clear. But it was more about the way you wrote the check. 

I find `if query_infos["metadata"].get("googleCloudStoragePathForLatestReport", None)` not explicit and hard to understand.

Reply:
Yes I understand, fixes are coming 👍